src/DataStore/DefaultDataStore.php

```diff
@@ -265,12 +265,12 @@ public function deleteResource($href, $resource)
      */
     public function executeRequest($method, UriInterface $uri, $body = '', array $options = [])
     {
-        $cacheManager = $cacheManager = Client::getInstance()->getCacheManager();
-        $cacheKey = $cacheManager->createCacheKey($uri);
+        // $cacheManager = $cacheManager = Client::getInstance()->getCacheManager();
```

Comment:
Could you just out right remove these lines instead of commenting them out? If we need to retrieve, we can just pull them from git history.

Reply:
Sure!